### PR TITLE
Ayushmishra/validate min SKU

### PR DESCRIPTION
**Pull request by novaturient95**

### CODE REVIEW

File: .github/workflows/assets-validation.yaml [hunk 1]
```diff
@@ -27,6 +27,10 @@ defaults:
   run:
     shell: bash
 
+permissions:
+  # Required for OIDC login to Azure
+  id-token: write
+
 jobs:
   validate:
     name: Validate
```
Comment (lykelly19) [inline .github/workflows/assets-validation.yaml:36]: below the line "runs-on: ubuntu-latest", add this line which should solve the access issue
```
    environment: Testing
```

File: .github/workflows/assets-validation.yaml [hunk 2]
```diff
@@ -47,12 +51,28 @@ jobs:
...
-    
+
+      - name: print
+        run: echo $client_id $tenant_id $subscription_id
+        env: 
+          client_id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant_id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription_id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+  
+      - name: Log in to Azure
```
Comment (lykelly19) [inline .github/workflows/assets-validation.yaml:62]: Could you put this log in section after install dependencies please?

File: .github/workflows/assets-validation.yaml (hunk 1 above)
Comment (novaturient95) [reply .github/workflows/assets-validation.yaml:36]: Thanks @lykelly19

File: .github/workflows/assets-validation.yaml (hunk 2 above)
Comment (novaturient95) [reply .github/workflows/assets-validation.yaml:56]: removed

**Pull request closed without merging**